### DISCUSSION TITLE
Which style config item powers the colour of the sub-tab line?

### DISCUSSION THREAD

**Discussion by PANstudio** [mbnuqw/sidebery#1396] [Q&A]
When I open a link in a tab, it spawns underneath the parent tab as a sort of sub-tab. In Sidebery, this is shown as follows (note the very faint line to the left of the child tab):

![image](https://github.com/mbnuqw/sidebery/assets/4048196/a8e8c961-667e-4f6b-8793-23cad73c6da7)

I can't discern from the style editor which config property controls the colour of that line, as I'd like to change it. Is this possible?

**Comment by piousdeer** [comment]
--tabs-normal-fg
